bigquery/tests/unit/test_table.py

```diff
@@ -1138,6 +1147,17 @@ def test_ctor(self):
         }
 
         table = self._make_one(resource)
+
+        if "creationTime" in resource:
```

Comment:
`creationTime` is in resource, as you are setting it above. This if statement is not needed. Just assert on the expected value. Same applies for `expirationTime`.

Reply:
Removed the If statements for creation an expiration, added asserts on expected value.